en/docs/install-and-setup/setup/reference/common-runtime-and-configuration-artifacts.md

```diff
@@ -53,25 +53,19 @@ The following are the artifacts used commonly in a WSO2 API Manager and API Mana
 
 ### Persistent Runtime Artifacts
 
-1. `<API-M_ANALYTICS_HOME>/repository/deployment/server` - Contains webapps, execution plans, event receivers, etc that are related to customizing WSO2 APIM Analytics during a deployment. Required for deploying a super tenant. It is sufficient to only share the Synapse configurations and execution plans between the nodes.
-
-2. `<API-M_ANALYTICS_HOME>/repository/data` - Contains the indexing files
-
-3. `<API-M_ANALYTICS_HOME>/repository/conf/analytics/` - My node id and shard allocation related data are stored in this directory.
+1. `<API-M_ANALYTICS_HOME>/wso2/worker/deployment/siddhi-files`
 
 !!! info
-        Shared Artifacts
+    Shared Artifacts
 
-    The following artifact can be shared among API Manager nodes
+    The following artifacts can be shared among API Manager nodes
 
-    1. `<API-M_HOME>/repository/deployment/server/synapse-configs`
-    2. `<API-M_HOME>/repository/deployment/server/executionplans`
+    1. <API-M_HOME>/repository/deployment/server/synapse-configs
+    2. <API-M_HOME>/repository/deployment/server/executionplans
 
```

Comment:
Shall we update this section as :
 "The following artifacts can be shared among APIM Analytics nodes"

 1. <API-M_ANALYTICS_HOME>/wso2/worker/deployment/siddhi-files

Reply:
Fixed